<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>实验6</string>
    </property>
    <addaction name="action1"/>
    <addaction name="separator"/>
    <addaction name="action2"/>
    <addaction name="separator"/>
    <addaction name="action3"/>
    <addaction name="separator"/>
    <addaction name="action4"/>
    <addaction name="separator"/>
    <addaction name="action5"/>
   </widget>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action1">
   <property name="text">
    <string>任务1：进度条模拟</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>任务2：查看区域网设备</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>任务3：长文本md5加密</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>任务4configparser模块的应用</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>任务5：json模块的应用</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
